seq2science/rules/alignment.smk

```diff
@@ -29,6 +29,8 @@ if config["aligner"] == "bowtie2":
             config["index"],
         shell:
             """
+            mkdir -p {output}
```

Comment:
Are these necessary? The new snakemake version does not make dirs anymore?

Reply:
When I added them it was indeed required. 
You could remove them to see if the bugfix releases changed that.